client/extensions/woocommerce/state/sites/promotions/README.md

```diff
@@ -0,0 +1,122 @@
+Promotions
+==========
+
+Promotions are a synthetic state construct that are actually created from one of two API objects: coupons, or products (with sale prices). Due to this, a promotion can have one of the following types:
+
+* `product_sale` (for a product with a sale price)
+* `percent` (coupon, matches coupon type)
+* `fixed_cart` (coupon, matches coupon type)
+* `fixed_product` (coupon, matches coupon type)
+
+## Promotions Object
+
+As Promotions only exist in memory state on the client at this point, the definition of a product is as follows:
```

Comment:
Did you mean `definition of a promotion`?

Reply:
Oops, yes I did, thanks!